extensions/amp-mathml/0.1/amp-mathml.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {Layout} from '../../../src/layout';
+import {cssstyle} from 'cssstyle';
+import {getIframe} from '../../../src/3p-frame';
+import { addParamsToUrl } from '../../../src/url';
+import { getDataParamsFromAttributes, removeElement } from '../../../src/dom';
+import { listenFor } from '../../../src/iframe-helper';
+
+export class AmpMathml extends AMP.BaseElement {
+
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {!Element} */
+    this.container_ = this.win.document.createElement('div');
+
+    /** @private {?HTMLIFrameElement} */
+    this.iframe_ = null;
+  }
+
+  /**
+  * @param {boolean=} opt_onLayout
+  * @override
+  */
+  preconnectCallback () {
+    this.preconnect.url('https://cdnjs.cloudflare.com');
+  }
+  /** @override */
+  buildCallback() {
+    const formula = this.element.getAttribute('formula');
+    if (!formula || '' === formula) {
```

Comment:
let's have 
```
  user().assert(
       data.formula,
       'The formula attribute is required for <amp-mathml> %s',
       data.element);
```
here instead.

Reply:
ok!

Reply:
that bit is in mathml.js, i removed the buildCallback entirely, it seems superfluous